lisp/i18n/ts/fr_FR/lisp.ts

```diff
@@ -94,17 +94,17 @@
     <message>
       <location filename="../../../ui/settings/app_pages/general.py" line="95"/>
       <source>Default layout</source>
-      <translation type="unfinished">Default layout</translation>
+      <translation>Disposition par défaut</translation>
```

Comment:
At another place, "layout" is translated as "agencement". I think "disposition" is better.